parasolr/query.py

```diff
@@ -158,6 +178,30 @@ def filter(self, *args, **kwargs) -> 'SolrQuerySet':
 
         return qs_copy
 
+    def facet(self, *args: str, **kwargs) -> 'SolrQuerySet':
+        """
+        Return a new SolrQuerySet with Solr faceting enabled and facet.field
```

Comment:
This doesn't seem as readable as I'd like (I think too technical / specific to what's going on underneath?). Maybe start with something like:
>  Request facets for the specified fields.
And then your existing description could follow.

Reply:
Done.